iothub/service/tests/Amqp/MockableAmqpCbsLink.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Framing;
+
+namespace Microsoft.Azure.Devices.Tests.Amqp
+{
+    public class MockableAmqpCbsLink
```

Comment:
```suggestion
   /// <summary>
   /// Since AmqpCbsLink from Amqp library is not overridable, this mockable class was created.
   /// </summary>
    public class MockableAmqpCbsLink
```